### DISCUSSION TITLE
STRSVR automatically closes multiple instances running at the same time after a given period

### DISCUSSION THREAD

**Discussion by Eaglecors** [tomojitakasu/RTKLIB#744] [General]
I have a server on which i run close to 10 instances of STRSVR with each collecting streams from a given GNSS station and transmitting the same to a specific caster. However these instances close without any reason after running for close to 4-6hours. What causes this to happen and how can i mitigate this problem?

**Comment by tomojitakasu** [comment]
Set the option "Inactive Timeout (ms)" to 0 (no inactive timeout) or a larger value.

As default, if the period of no data exceeds 10 seconds, STRSVR disconnects the 
connection and re-try to connect the caster after the time specified as
"Reconnect Interval".
